l2geth/rollup/gas_price_updater_test.go

```diff
@@ -0,0 +1,82 @@
+package rollup
+
+import (
+	"fmt"
+	"testing"
+)
+
+type MockEpoch struct {
+	timestamp float64
+	numBlocks uint64
+}
+
+// WIP
```

Comment:
Should add more information about this, why is it WIP and what's the next step?

Reply:
Ah yes this whole PR should probably be WIP until I finish adding tests to the gasUpdater. Let me change it to draft real quick -- IMO these tests are not quite up to snuff

Reply:
Added a bunch of tests & removed the WIP!